connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverterConfig.java

```diff
@@ -76,4 +93,14 @@ public boolean schemasEnabled() {
     public int schemaCacheSize() {
         return getInt(SCHEMAS_CACHE_SIZE_CONFIG);
     }
+
+    /**
+     * Get the serialization format for decimal types.
+     *
+     * @return the decimal serialization format
+     */
+    public DecimalFormat decimalFormat() {
+        return DecimalFormat.valueOf(getString(DECIMAL_FORMAT_CONFIG).toUpperCase(Locale.ROOT));
```

Comment:
This is really expensive to do every time.

Reply:
Resolved using your second suggestion.